src/diem/offchain/__init__.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) The Diem Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+""" This package provides data structures and utilities for implementing Diem Offchain API Service.
+
+See [Diem Offchain API](https://dip.diem.com/lip-1/) for more details.
+
+"""
+
+from .types import (
+    AbortCode,
+    CommandType,
+    CommandResponseStatus,
+    OffChainErrorType,
+    ErrorCode,
+    PaymentCommandObject,
+    CommandRequestObject,
+    CommandResponseObject,
+    OffChainErrorObject,
+    PaymentObject,
+    PaymentActorObject,
+    PaymentActionObject,
+    Status,
+    StatusObject,
+    KycDataObjectType,
+    NationalIdObject,
+    AddressObject,
+    KycDataObject,
+    FieldError,
+    InvalidOverwriteError,
+    new_payment_object,
+    new_payment_request,
+    replace_payment_actor,
+    reply_request,
+    individual_kyc_data,
+    entity_kyc_data,
+    to_json,
+    from_json,
+    validate_write_once_fields,
+)
+from .http_header import X_REQUEST_ID, X_REQUEST_SENDER_ADDRESS
+from .action import Action
+from .error import command_error, protocol_error, Error
+from .command import Command
+from .payment_command import PaymentCommand
+from .client import Client, CommandResponseError
+
+from . import jws, http_server, state, payment_state
+from .. import identifier, txnmetadata
+
+import typing
+
+
+def travel_rule_metadata(payment: PaymentObject, hrp: str) -> typing.Tuple[bytes, bytes]:
```

Comment:
why is this defined here?

Reply:
convert payment: PaymentObject into travel rule metadata, 

offchain API is higher level comparing with DIP-4 implementation, so this func calls to txnmetadata.travel_rule to create data

Reply:
this is removed and merged into PaymentCommand related functions.